modules/special-rules/pages/tyranid-objectives.adoc

```diff
@@ -0,0 +1,16 @@
+= Tyranid Objectives
+
+Always ready for a krumpin, Orks will often charge into melee before the enemy has even realised that the Orks are close enough to charge them!
```

Comment:
This is definitely in the wrong place... :-D

Reply:
Yeah... 🤣

That was going to be swapped out for something about the Nids only caring about biomass rather than holding ground etc.

```suggestion
The Hive Mind hungers only for biomass and genetic material, and doesn't hold ground or capture objectives.
```